<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Background{
background-color:qlineargradient(spread:pad, x1:0.04, y1:0.0625, x2:1, y2:1, stop:0 rgba(150, 162, 207, 255), stop:1 rgba(235, 242, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 36pt &quot;Seagull&quot;;</string>
    </property>
    <property name="text">
     <string>Inventory Bot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="target">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
color:rgb(193, 193, 193);
font: 28pt &quot;Seagull&quot;;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(216, 211, 255, 255), stop:1 rgba(255, 255, 255, 255))
</string>
    </property>
    <property name="text">
     <string>Target</string>
    </property>
   </widget>
   <widget class="QPushButton" name="amazon1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
color:rgb(193, 193, 193);
font: 28pt &quot;Seagull&quot;;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(216, 211, 255, 255), stop:1 rgba(255, 255, 255, 255))
</string>
    </property>
    <property name="text">
     <string>Amazon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0,0,0,0);
font: 22pt &quot;Seagull&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;- Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
